src/panels/SkyBrowserPanel/WorldWideTelescope/WwtProvider/types.d.ts

```diff
@@ -0,0 +1,60 @@
+interface HideChromeMessage {
+  event: 'modify_settings';
+  settings: [['hideAllChrome', true]];
+  target: 'app';
+}
+
+interface LoadCollectionMessage {
+  event: 'load_image_collection';
+  url: string;
+  loadChildFolders: true;
+}
+
+interface BorderRadiusMessage {
+  event: 'set_border_radius';
+  data: number;
+}
+
+interface ColorMessage {
+  event: 'set_background_color';
+  data: number[];
+}
+
+interface AimMessage {
+  event: 'center_on_coordinates';
+  ra: number;
+  dec: number;
+  fov: number;
+  roll: number;
```

Comment:
Do we want tot add acomment for each of these ,mentioning what types of values are expected?  I.e. whether the unit is in degrees/radians for example. I think that's good practice at least when dealing with angles